packages/grpc-js/src/server.ts

```diff
@@ -227,4 +229,108 @@ export class Server {
   addHttp2Port(): void {
     throw new Error('Not yet implemented');
   }
+
+  private _setupHandlers(): void {
+    if (this.http2Server === null) {
+      return;
+    }
+
+    this.http2Server.on(
+        'stream',
+        (stream: http2.ServerHttp2Stream,
+         headers: http2.IncomingHttpHeaders) => {
+          if (this.started !== true) {
```

Comment:
`this.started` is a boolean. You don't have to explicitly compare it with `true`.

Reply:
This seems to be a common pattern with the patch.